src/Microsoft.ApplicationInsights.AspNet/Extensions/ApplicationInsightsExtensions.cs

```diff
@@ -181,5 +184,22 @@ private static void InitializeModulesWithConfiguration(IServiceProvider serviceP
                 service.Initialize(configuration);
             }
         }
+
+        private static void AddServerTelemetryChannelAndSamplingForFullFramework(IServiceProvider serviceProvider, TelemetryConfiguration configuration, bool DisableSampling)
```

Comment:
I'm wondering if instead of having a flag and making an internal call for using adaptive sampling if we should just have the users make a call or pass the class they want to use as a parameter.